zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryResponseMapper.java

```diff
@@ -65,4 +68,51 @@ public static Either<ProblemDetail, ProcessInstanceItem> toProcessInstance(
             .startDate(p.startDate())
             .endDate(p.endDate()));
   }
+
+  public static UserTaskSearchQueryResponse toUserTaskSearchQueryResponse(
+      final SearchQueryResult<UserTaskEntity> result) {
+    final var response = new UserTaskSearchQueryResponse();
+    final var total = result.total();
+    final var sortValues = result.sortValues();
+    final var items = result.items();
+
+    final var page = new SearchQueryPageResponse();
+    page.setTotalItems(total);
+    response.setPage(page);
+
+    if (sortValues != null) {
+      page.setLastSortValues(Arrays.asList(sortValues));
+    }
+
+    if (items != null) {
+      response.setItems(toUserTasks(items).get());
+    }
+
+    return response;
+  }
+
+  public static Either<ProblemDetail, List<UserTaskItem>> toUserTasks(
+      final List<UserTaskEntity> tasks) {
+    return Either.right(
```

Comment:
🔧 This never returns a `Either.Left` so you can simply return the success type and simply the code.